demo/src/javascripts/components/home.js

```diff
@@ -35,6 +36,18 @@ class Home extends Component {
     this._onScroll();
     this._resizeMap();
 
+    this._stats = new Stats();
+    this._stats.showPanel(0);
+    this.refs.fps.appendChild(this._stats.dom);
+
+    const calcFPS = () => {
+      this._stats.begin();
```

Comment:
@apercu - yes, good first step. To get really correct results, we may need to hook it into the DeckGL rendering loop, but maybe this is good enough for now. Let's merge it so I can test.